images/spark-operator/config/main.tf

```diff
@@ -1,17 +1,32 @@
 variable "extra_packages" {
   description = "Additional packages to install."
   type        = list(string)
-  default     = ["spark-operator"]
+  default     = ["spark", "spark-operator"]
```

Comment:
Can we move `spark` to the list of packages below? Otherwise it'll get overridden with just `spark-operator` inside the private build.

Reply:
Hey @pdeslaur, there are some circumstances where we need to override both packages, so this is intentional. It does mean both need to be passed in when / if this module is invoked elsewhere